test/Psr/CacheItemPool/CacheItemTest.php

```diff
@@ -106,4 +110,32 @@ public function testExpiresAfterInvalidThrowsException(): void
         $item = new CacheItem('key', 'value', true);
         $item->expiresAfter([]);
     }
+
+    public function testExpiresAfterStartsExpiringAfterMethodCall(): void
+    {
+        $item = new CacheItem('key', 'value', false);
+        $item = $item->expiresAfter(10);
+        sleep(1);
+        self::assertEquals(9, $item->getTtl());
+    }
+
+    public function testClockProvidedDoesNotContainUTCTimeZone(): void
+    {
+        $item = new CacheItem(
+            'foo',
+            null,
+            false,
+            new SystemClock(new DateTimeZone('Europe/Berlin'))
```

Comment:
In tests it usually makes more sense to use the `FrozenClock` instead of the actual one.